### DISCUSSION TITLE
Help to Restore Deleted Gists

### DISCUSSION THREAD

**Discussion by BingoKingo** [community/community#29171] [Other Feature Feedback, Questions, & Ideas] [answered]
I delete an important app config backup in gist, and I don’t remember the link.

Anyone could help me to restore deleted gists? I only find out a way to restore deleted repo…but not gist, crying :(

**Comment by BingoKingo** [comment, marked as answer]
Contact with support…

**Reply by Orlando-Gama** [reply]
some weeks ago l deleted a main branch under repository "alx-system_engineering-devops" so l had a lot directories and files under it. so this has affected all my servers, and in my gitbut under that repo, all the projects cant be seen because most of the projects l did was in that main branch so how can you help me restore it
